### PR TITLE
Proposal for a decision process to be followed.

### PR DESCRIPTION
**Pull request by p0nce**
Hello, I propose the following rules to govern the Working Group.

Do you agree?

Please read the content of this file, this PR has to follow it itself which avoid an annoying "bootstrapping" exception.

### CODE REVIEW

File: decision-process.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+GUI libraries are notoriously complex to design and are prone to easy disapproval.
+
+The DlangGraphicsWG initiative comes from a shared understanding disagreement must be
+reified and talked upon, rather than hidden behind individual initiatives.
+
+
+1. The output of this Work Group is every Markdown .md files in the Organization's repositeries.
+   The wiki is only informal.
+
+2. Members of the Github organization DlangGraphicsWG have no special right,
```
[the remark this answers is not in the document]
Comment (p0nce) [reply decision-process.md]: fixed

File: decision-process.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Decision Process
+
+GUI libraries are notoriously complex to design and are prone to easy disapproval.
+
+The DlangGraphicsWG initiative comes from a shared understanding disagreement must be
+reified and talked upon, rather than hidden behind individual initiatives.
+
+
+1. The output of this Work Group is every Markdown .md files in the Organization's repositeries.
```
[the remark this answers is not in the document]
Comment (p0nce) [reply decision-process.md]: fixed

**Comment by rikkimax** [comment]
It's probably a little overkill, but no matter.

**Comment by p0nce** [comment]
Merged! If democracy doesn't suit you, you can democratically propose changes to avoid it!

**Pull request merged**